CriteoPublisherSdk/Sources/DataConsent/CR_GdprVersion.h

```diff
@@ -61,6 +61,30 @@ extern NSString *const CR_GdprConsentStringForTcf1_1Key;
  */
```

Comment:
I am curious what is the difference between `CR_GdprVersion` and `CR_Gdpr`? I see that both declare the same methods, such as: `isVendorConsentGiven` and `hasVendorLegitimateInterest `

Reply:
This is how TCF v1 & v2 overlap has been handled. One implementation per version. Switching logic being on top.